### PR TITLE
Fix snake conversion 

### LINKED ISSUE #2393: Ansible and inserted underscores in extravars
## Bug Report

**What did you do?**
I created a cr with, in the spec the "create_k8s_namespace" key

**What did you expect to see?**
I expect to see "_create_k8s_namespace_" as an extravars.
Confirmed reading _/tmp/ansible-operator/runner/XYZ/v1alpha1/ASD/default/test-2/env/extravars_

**What did you see instead? Under which circumstances?**
I see "_create_k_8_s_namespace_" (with inserted underscores)

**Environment**
* operator-sdk version: 1.14.0
* Kubernetes version information: v1.15.4-gke.22
* Kubernetes cluster kind: gke

* Are you writing your operator in ansible, helm, or go?
ansible

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
Fix snake conversion

**Motivation for the change:**
Closes #2393

### CODE REVIEW

File: pkg/ansible/paramconv/paramconv_test.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+			want: "var",
+		},
+		{
+			name: "should convert to Snake when is only numbers",
+			args: args{"888"},
+			want: "888",
+		},
+		{
+			name: "should convert to Snake when has numbers",
+			args: args{"k8s_var"},
```
Comment (fabianvf) [inline pkg/ansible/paramconv/paramconv_test.go:147]: This should be camelCase, eg `k8sVar`, right?

Comment (camilamacedo86) [reply pkg/ansible/paramconv/paramconv_test.go:147]: it is a unit test for the snake only,
We can add this test in the testToCamel added here to check as well. I will do.

Comment (fabianvf) [reply pkg/ansible/paramconv/paramconv_test.go:151]: But the toSnake tests should be testing from camelCase to snake_case I think, I only see tests that are doing snake_case -> snake_case

**Comment by jmrodri** [comment]
The tests in `TestToSnake` aren't that extensive. We're passing in `var`, `888`, `k8s_var`, `_k8s_var` and `k8s var`. Three of these are already snake case. `var`, `k8s_var` and `_k8s_var` are all no-op test cases since they are already in snake_case.

I think better test would be:

* ThisShouldHaveUnderscores -> this_should_have_underscores
* k8sVar -> k8s_var
* _CanYou_Handle_mixedVars -> _can_you_handle_mixed_vars
* this_should_be_a_noop -> this_should_be_a_noop

**Comment by camilamacedo86** [comment]
Hi @jmrodri. 

Really tks for the input. Added the missing scenarios.

**Pull request merged**